game-core/src/main/java/games/strategy/triplea/ui/PoliticsPanel.java

```diff
@@ -126,8 +126,7 @@
                       0));
               final JButton noActionButton =
                   new JButton(
-                      SwingAction.of(
-                          "No Actions", event -> politicalChoiceDialog.setVisible(false)));
+                      SwingAction.of("Done", event -> politicalChoiceDialog.setVisible(false)));
```

Comment:
Is there a 'close' listener attached anywhere? The other buttons now uniformly have "shift+d" mapped to them, this would be missing that.

Reply:
Added.

Reply:
Please double check these via manual testing if you have not already. We have an unresovled bug where key bindings are always active, I suspect typing in chat "D" would close the politics window, but that is an existing problem we need to solve.

Reply:
This code is applying to the Politics dialog, which is modal. While it is active, it's not possible to type D in chat, because the main window can't receive focus.

I can confirm that typing D in chat activates the shortcut in the main window when this dialog is not open, but this is not related to my change.

Reply:
Thanks for double checking that :smiley: